app/models/rummager_search.rb

```diff
@@ -32,6 +32,22 @@ def start
     search_result["start"]
   end
 
+  def organisations
+    @_organisations ||= search_result.dig('aggregates', 'organisations', 'options').map do |option|
```

Comment:
Ah, this is what I was missing! Awesome 👍

Reply:
😄